### PR TITLE
fix: "Error: no such option: --backup"

### PR DESCRIPTION
**Pull request by ccfiel**
https://discuss.erpnext.com/t/bench-update-fails-due-to-no-bench-backup-option/58697

What type of a PR is this?

- [ ] Changes to Existing Features
- [ ] New Feature Submissions
- [x] Bug Fix
- [ ] Breaking Change (include change in API behaviours, etc.)

---

> Please provide enough information so that others can review your pull request:

---

> Explain the **details** for making this change. What existing problem does the pull request solve? The following checklist could help

- [ ] Have you followed the guidelines in our Contributing document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
- [x] Have you lint your code locally prior to submission?
- [x] Have you successfully run tests with your changes locally?
- [ ] Does your commit message have an explanation for your changes and why you'd like us to include them?
- [ ] Docs have been added / updated
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Did you modify the existing test cases? If yes, why?

---

> Screenshots/GIFs

### CODE REVIEW

File: bench/commands/update.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 @click.option('--reset', is_flag=True, help="Hard resets git branch's to their new states overriding any changes and overriding rebase on pull")
 def update(pull, patch, build, bench, requirements, restart_supervisor, restart_systemd, no_backup, force, reset):
 	from bench.utils import update
-	update(pull=pull, patch=patch, build=build, bench=bench, requirements=requirements, restart_supervisor=restart_supervisor, restart_systemd=restart_systemd, backup= not no_backup, force=force, reset=reset)
+	update(pull=pull, patch=patch, build=build, bench=bench, requirements=requirements, restart_supervisor=restart_supervisor, restart_systemd=restart_systemd, backup= no_backup, force=force, reset=reset)
```
Comment (gavindsouza) [inline bench/commands/update.py]: For an instance when backup isn't needed, `backup` should be set to `False`
when `--no-backup` flag is used, `no_backup` is set to `True` so `backup = not no_backup` is the proper expression

Comment (ccfiel) [reply bench/commands/update.py]: @gavindsouza but in present code I tested if you put --no-backup it will backup that's why I change the logic.

Comment (gavindsouza) [reply bench/commands/update.py]: This is from testing your branch:

Screenshot 2020-03-04 at 8 39 26 AM

Comment (ccfiel) [reply bench/commands/update.py]: @gavindsouza ok I change the code again

**Pull request merged**